### PR TITLE
feat: support attributes to forward to ProseMirror editor

### PR DESCRIPTION
**Pull request by clemdesign**
This PR contains:

- [ ] bugfix
- [x] feature
- [ ] refactor
- [ ] tests
- [x] documentation
- [ ] others

### Breaking Changes?

- [ ] yes
- [x] no

### Checklist

- [x] commit messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
  - A document related commit is prefixed "docs:"
- [x] docs have been added / updated (for bug fixes / features)

### Describe Your Changes

Refer to discussion: https://github.com/sibiraj-s/ngx-editor/discussions/365

### Does this PR affects any existing issues?

- [ ] yes
- [x] no

### CODE REVIEW

File: projects/ngx-editor/src/lib/Editor.ts [hunk 1]
```diff
@@ -21,6 +22,7 @@ interface Options {
   schema?: Schema;
   plugins?: Plugin[];
   nodeViews?: EditorProps['nodeViews'];
+  enterKeyHint?: EnterKeyHint;
```
Comment (sibiraj-s) [inline projects/ngx-editor/src/lib/Editor.ts:25]:
```suggestion
  attributes?: Record<string, string>;
```

we can get entire attributes here for forward it to prosemirror.

Comment (clemdesign) [reply projects/ngx-editor/src/lib/Editor.ts:25]: OK, why not. I will change the feature.

Comment (clemdesign) [reply projects/ngx-editor/src/lib/Editor.ts]: Done.

**Comment by sibiraj-s** [comment]
@clemdesign changes looks fine. Please revert the style changes introduced.

**Comment by clemdesign** [comment]
@sibiraj-s done.

**Review by sibiraj-s — APPROVED** [review]

**Comment by sibiraj-s** [comment]
Published in v11.

**Comment by clemdesign** [comment]
Thank you 👍

**Comment by clemdesign** [comment]
@sibiraj-s 
This is possible to apply these changes to v9.X.X ?

I currently use Angular 10 and it's not planned to upgrade version now.

**Pull request merged**